From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Evgeny Karpov <evgeny.karpov@microsoft.com>
Date: Mon, 4 Nov 2024 14:47:51 +0100
Subject: [PATCH 01/29] aarch64: Bypass some warnings in MinGW

The patch bypasses missing prototypes warnings in MinGW until it is
fixed.

libgcc/ChangeLog:

	* config/aarch64/t-mingw: Update.

(cherry picked from commit d6ccc4ddd3a7c9737a30dc8927d6835be2f0656e)
---
 libgcc/config/aarch64/t-mingw | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libgcc/config/aarch64/t-mingw b/libgcc/config/aarch64/t-mingw
index a72a2f70177..27d28d42c91 100644
--- a/libgcc/config/aarch64/t-mingw
+++ b/libgcc/config/aarch64/t-mingw
@@ -1,2 +1,2 @@
-# Bypass hidden attribute warnings in MinGW until it is implemented
-LIBGCC2_CFLAGS += -Wno-error=attributes
+# Bypassing hidden attribute warnings in MinGW until it is implemented
+LIBGCC2_CFLAGS += -Wno-error=attributes -Wno-error=missing-prototypes
